style.rmd

```diff
@@ -175,11 +192,31 @@ When indenting your code, use two spaces. Never use tabs or mix tabs and spaces.
 The only exception is if a function definition runs over multiple lines. In that case, indent the second line to where the definition starts:
 
 ```{r, eval = FALSE}
+# Good
+do_something_very_complicated("that", requires, many, arguments,
```

Comment:
But those are two different styles of function - the first is `...` style like `paste()`, and the second is one where arguments are named.